Form1.html

```diff
@@ -36,14 +34,10 @@
     <table>
        <tr>
          <td id="Box1">
-            <h3>Get Item from Local Storage</h3>
-            <span>Enter key to retrieve item: </span><input type="text" id="itemls">
-            <input id="getitemls" type="button" value="Retrieve Item">
-        </td>
-        <td id="Box2">
-            <h3>Get Item from Session Storage</h3>
-            <span>Enter key to retrieve item: </span><input type="text" id="itemss">
-            <input id="getitemss" type="button" value="Retrieve Item">
+            <h3>Get Item from Local/Session Storage</h3>
+            <span>Enter key to retrieve item: </span><input type="text" id="getitem">
+            <input id="getitem" type="button" value="Retrieve Item from Local Storage" onclick="getSingleItem.call(local)">
```

Comment:
onclick callback should be the method which returned from bind function.
In JS: 
var getFromLocalStorage = getSingleItem.bind(local);
In HTML:
onclick="getFromLocalStorage()"